spikeextractors/tests/test_neuroscopesortingextractor.py

```diff
@@ -0,0 +1,67 @@
+import unittest
+import numpy as np
+import spikeextractors as se
+import os
+import tempfile
+import shutil
+from pathlib import Path
+
+class TestNeuroscopeSortingExtractors(unittest.TestCase):
```

Comment:
For testing, could you remove this and add a test in the [`test_extractors.py`](https://github.com/SpikeInterface/spikeextractors/blob/master/spikeextractors/tests/test_extractors.py#L376)

We have several tests there that use the write funciton to save an extractor to file, and then loads it back and checks that everything is ok.

Reply:
So you want all tests for the NeuroscopeSortingExtractor functionality in the test_extractors.py, as opposed to a separate file like the NpzSortingExtractor is?

Reply:
Yes :) I will also move the NpzSortingExtractor tests there (sorry about the confusion). 

The NumpyExtractors, instead, are a special case because they are in-memory and not bound to a file, so they should have their own tests.

Reply:
Unlike when the test was in a separate file, I can't run the test_....py locally without getting a bunch of permission errors for the temp directories from all the other extractors, so I'm relying on the travis-CI for that. If you get notifications each time it fails and I do a re-commit, I apologize for the spam.

Reply:
No worries!